official/utils/logs/metric_hook_test.py

```diff
@@ -33,49 +34,35 @@ class LoggingMetricHookTest(tf.test.TestCase):
   def setUp(self):
     super(LoggingMetricHookTest, self).setUp()
 
-    class MockMetricLogger(object):
-
-      def __init__(self):
-        self.logged_metric = []
-
-      def log_metric(self, name, value, unit=None, global_step=None,
-                     extras=None):
-        self.logged_metric.append({
-            "name": name,
-            "value": float(value),
-            "unit": unit,
-            "global_step": global_step,
-            "extras": extras})
-
     self._log_dir = tempfile.mkdtemp(dir=self.get_temp_dir())
-    self._logger = MockMetricLogger()
+    self._logger = logger.MockBenchmarkLogger()
 
   def tearDown(self):
     super(LoggingMetricHookTest, self).tearDown()
     tf.gfile.DeleteRecursively(self.get_temp_dir())
 
   def test_illegal_args(self):
-    with self.assertRaisesRegexp(ValueError, 'nvalid every_n_iter'):
-      metric_hook.LoggingMetricHook(tensors=['t'], every_n_iter=0)
-    with self.assertRaisesRegexp(ValueError, 'nvalid every_n_iter'):
-      metric_hook.LoggingMetricHook(tensors=['t'], every_n_iter=-10)
-    with self.assertRaisesRegexp(ValueError, 'xactly one of'):
+    with self.assertRaisesRegexp(ValueError, "nvalid every_n_iter"):
```

Comment:
nit: It's intended to miss the first letter here?

Reply:
Yes. I was just changing quote styles while I was here, but often people will leave out the first letter of a regex test to avoid having to deal with whether the sentence was capitalized or not.